src/panels/config/zha/zha-add-group-page.ts

```diff
@@ -0,0 +1,202 @@
+      >
+        <ha-config-section .isWide=${!this.narrow}>
+          <p slot="introduction">
+            ${this.hass.localize(
+              "ui.panel.config.zha.groups.create_group_details"
+            )}
+          </p>
+          <paper-input
+            type="string"
+            .value="${this._groupName}"
+            @value-changed=${this._handleNameChange}
+            placeholder="${this.hass!.localize(
+              "ui.panel.config.zha.groups.group_name_placeholder"
+            )}"
+          ></paper-input>
+
+          <div class="header">
+            ${this.hass.localize("ui.panel.config.zha.groups.add_members")}
+          </div>
+
+          <zha-devices-data-table
+            .hass=${this.hass}
+            .devices=${this.devices}
+            .narrow=${this.narrow}
+            .selectable=${true}
+            @selection-changed=${this._handleAddSelectionChanged}
+            class="table"
+          >
+          </zha-devices-data-table>
+
+          <div class="paper-dialog-buttons">
+            <mwc-button
+              .disabled="${!this._canAdd || this._processingAdd}"
+              @click="${this._createGroup}"
+              class="button"
+            >
+              <paper-spinner
+                ?active="${this._processingAdd}"
+                alt="${this.hass!.localize(
+                  "ui.panel.config.zha.groups.creating_group"
+                )}"
+              ></paper-spinner>
+              ${this.hass!.localize(
+                "ui.panel.config.zha.groups.create"
+              )}</mwc-button
+            >
+          </div>
+        </ha-config-section>
+      </hass-subpage>
+    `;
+  }
+
+  private async _fetchData() {
+    this.devices = await fetchGroupableDevices(this.hass!);
+  }
+
+  private _handleAddSelectionChanged(ev: CustomEvent): void {
+    const changedSelection = ev.detail as SelectionChangedEvent;
+    const entity = changedSelection.id;
+    if (changedSelection.selected) {
+      this._selectedDevicesToAdd.push(entity);
+    } else {
+      const index = this._selectedDevicesToAdd.indexOf(entity);
+      if (index !== -1) {
+        this._selectedDevicesToAdd.splice(index, 1);
+      }
+    }
+    this._selectedDevicesToAdd = [...this._selectedDevicesToAdd];
+  }
+
+  private async _createGroup(): Promise<void> {
+    this._processingAdd = true;
+    const group: ZHAGroup = await addGroup(
+      this.hass,
+      this._groupName,
+      this._selectedDevicesToAdd
+    );
+    this._selectedDevicesToAdd = [];
+    this._canAdd = false;
+    this._processingAdd = false;
+    this._groupName = "";
+    navigate(this, `/config/zha/group/${group.group_id}`, true);
+  }
+
+  private _handleNameChange(ev: PolymerChangedEvent<string>) {
+    const target = ev.currentTarget as PaperInputElement;
+    if (target.value) {
+      this._groupName = target.value;
+      if (target.value.length > 0) {
+        this._canAdd = true;
+      }
+    }
```

Comment:
The value of an input element is always a string.
```ts
this._groupname = target.value;
```

Reply:
fixed